src/trace_processor/perfetto_sql/stdlib/android/memory/kswapd.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Copyright 2025 The Android Open Source Project
+--
+-- Licensed under the Apache License, Version 2.0 (the 'License');
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     https://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an 'AS IS' BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- Breakdown of kswapd duration on each cpu.
+CREATE PERFETTO VIEW android_kswapd_cpu_breakdown (
+  -- cpu
```

Comment:
Please also expose the ucpu column of sched for situations where we have multi-machine tracing.